posydon/utils/common_functions.py

```diff
@@ -228,6 +228,20 @@ def roche_lobe_radius(q, a_orb=1):
     .. [1] Eggleton, P. P. 1983, ApJ, 268, 368
 
     """
+
+    if isinstance(a_orb, np.ndarray):
+        Pwarn("Trying to compute RL radius for binary with no separation", "EvolutionWarning")
```

Comment:
I'd more go for a `InappropriateValueWarning` here.
Additionally, I don't see a good reason to set here an input value to `nan`. This will cause the output to be always `nan`, hence we can directly return a `nan`.